lib/blockscore/base.rb

```diff
@@ -60,8 +60,16 @@ def persisted?
       !id.nil? && !deleted?
     end
 
+    def deleted?
```

Comment:
@dkubb apologies - forgot to address your question of `deleted?` coverage. The answer is that I have been looking at the mutation of Base, but it is very slow and large. There are 672 mutations. Since most is private I need to get a gameplan for it together.